src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -1,70 +1,57 @@
 package com.kodilla.ecommercee.controller;
 
+import com.kodilla.ecommercee.domain.Product;
 import com.kodilla.ecommercee.dto.ProductDto;
+import com.kodilla.ecommercee.exception.ProductNotFoundException;
+import com.kodilla.ecommercee.mapper.ProductMapper;
+import com.kodilla.ecommercee.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.math.BigDecimal;
-import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 @CrossOrigin("*")
 @RequestMapping("/products")
 public class ProductController {
-
-    List<ProductDto> testProductList = new ArrayList<ProductDto>() {
-        {
-            add(new ProductDto(1L, "kurtka zimowa", "Pellentesque(...)", new BigDecimal(100), 1L));
-            add(new ProductDto(2L, "płaszcz", "Pellentesque(...)", new BigDecimal(150), 1L));
-            add(new ProductDto(3L, "buty", "Pellentesque(...)", new BigDecimal(100), 4L));
-            add(new ProductDto(4L, "rękawiczki", "Pellentesque(...)", new BigDecimal(50), 2L));
-        }
-    };
+    @Autowired
+    ProductService productService;
+    @Autowired
+    ProductMapper productMapper;
 
     @GetMapping
     public ResponseEntity<List<ProductDto>> getProducts() {
-        return ResponseEntity.ok(testProductList);
+        List<Product> productList = productService.getAllProducts();
+        return ResponseEntity.ok(productMapper.mapToProductDtoList(productList));
```

Comment:
It's totally a matter of style and what one considers readable, but you can throw the list generation directly into the return.